example/eventsource-polyfill.js

```diff
@@ -12056,4 +12080,4 @@ module.exports = function isBuffer(arg) {
 }
 
 /***/ })
-/******/ ]);
+/******/ ]);
```

Comment:
newline

Reply:
I'm not sure if it was manually edited before, but it generated this way.